### DISCUSSION TITLE
Migrated from v3.4.7 to 3.5.1 I cannot login

### DISCUSSION THREAD

**Discussion by kuratowsky** [mastodon/mastodon#18029] [Q&A]
Dear all, 
I followed (I think) all the migration steps described in https://github.com/mastodon/mastodon/releases/tag/v3.5.0 and https://github.com/mastodon/mastodon/pull/16947 
My installation is dockerized.
When I open the website, it seems the admin username does not exist.

![image](https://user-images.githubusercontent.com/482854/163048047-f6c0297e-deea-4ae7-b0cf-bd3877d19a9d.png)

Reading the DB logs, I've got a lot of "ERROR: duplicate key value violates unique constraints "anything"".

If I search my admin username in the log, it appears.

How can access to the database and re-create my username/pass in order to gain control again?

Thank you very much in advance!

Cheers

**Comment by Gargron** [comment]
cc @shleeable

**Reply by shleeable** [reply]
@kuratowsky The errors in the DB seem to be related to the export or import going badly... So I'd recommend stopping everything
and taking a fresh backup. Please follow the vacuum instructions as people had similar issues that were resolved by this additional step
